docs/features.md

```diff
@@ -52,6 +52,16 @@ The `Disruption` resource uses [label selectors](https://kubernetes.io/docs/conc
 
 **NOTE:** If you are targeting pods, the disruption must be created in the same namespace as the targeted pods.
 
+### DynamicTargeting
+
+By default, targeting is limited to a single target selection at the disruption's creation. It allows for more controlled disruption impact and propagation, as the targets will never change and canbe compensated in case of issue. Its major limit is not being able to follow targets through deployments/rollouts.
+By setting the `DynamicTargeting` flag to `true` in the Disruption's yaml spec, you activate a constant re-targeting for this disruption. This means at any given time, any target within the selector's scope will be added to the target list and be disrupted.
+This is a feature to use with care, as it can quickly get out of control: per example, a disruption targeting 100% of an application's pod will affect all existing **and** future pods which can appear once the disruption started. As long as this 100% disruption exists, there will be no spared pod.
+Even a 50%-set disruption, if the disruption gets targeted pods to die, will constantly re-target 50% of all selector-fitting pods and end-up killing every pods.
```

Comment:
```suggestion
Even a 50%-set disruption, if the disruption gets targeted pods to die, will constantly re-target 50% of all selector-fitting pods and end up killing every pod.
```

Is that true? I think I know what you mean here, but as long as k8s is scheduling new pods, we'll keep at least half of them up, so we dont really kill "every" pod, though we may end up killing any given pod eventually as the disruption/rescheduling continues

Reply:
Yes that's accurate; though that will most likely create a race condition and it'll be a weird balance to maintain.
I'll test this locally with a pod-killing disruption.